### PR TITLE
ticdc: clarify rowid HandleKey meaning

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

clarify rowid HandleKey meaning

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/8156
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -579,7 +579,7 @@ worker-num = 16
 # The dispatcher rules are as follows:
 # - default: When multiple unique indexes (including the primary key) exist or the Old Value feature is enabled, events are dispatched in the table mode. When only one unique index (or the primary key) exists, events are dispatched in the rowid mode.
 # - ts: Use the commitTs of the row change to create Hash and dispatch events.
-# - rowid: Use the name and value of the selected HandleKey column to create Hash and dispatch events.
+# - rowid: Use the primary key or the unique index column name and column value of the table to create Hash and dispatch events.
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:582]:
```suggestion
# - rowid: Use the row name and row value of the primary key or the unique index of the table to create Hash and dispatch events.
```
参数是 rowid，所有这里的列应该是 row?另外是主键的列名和列值+唯一索引的列名和列值？

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -579,7 +579,7 @@ worker-num = 16
 # The dispatcher rules are as follows:
 # - default: When multiple unique indexes (including the primary key) exist or the Old Value feature is enabled, events are dispatched in the table mode. When only one unique index (or the primary key) exists, events are dispatched in the rowid mode.
 # - ts: Use the commitTs of the row change to create Hash and dispatch events.
-# - rowid: Use the name and value of the selected HandleKey column to create Hash and dispatch events.
+# - index-value: Use the row name and row value of the primary key or the unique index of the table to create Hash and dispatch events.
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:582]:
```suggestion
# - index-value: Use the value of the primary key or the unique index of the table to create Hash and dispatch events.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**